### ISSUE TITLE
cannot create new Boolean Constructor

### ISSUE THREAD

**Issue by arunrajdpm** [freeCodeCamp/freeCodeCamp#15240]
Challenge [Falsy Bouncer](https://www.freecodecamp.com/challenges/falsy-bouncer#?solution=%0Afunction%20bouncer(arr)%20%7B%0A%20%20%2F%2F%20Don't%20show%20a%20false%20ID%20to%20this%20bouncer.%0A%20%20return%20arr%3B%0A%7D%0A%0Abouncer(%5Bfalse%2C%20null%2C%200%2C%20NaN%2C%20undefined%2C%20%22%22%5D.filter(function(value)%7B%0A%20%0A%20%20console.log(null%20%3D%3D%3D%20null)%3B%0A%20%20%0A%20%20if(!isNaN(value))%0A%20%20if(value%20%3D%3D%3D%20%22%22%20%7C%7C%20value%20%3D%3D%3D%20false%20%20%7C%7C%20value%20%3D%3D%3D%20null%20%7C%7C%20value%20%3D%3D%3D%20undefined%20%7C%7C%20value%20%3D%3D%3D%200)%0A%20%20%20%20%20%20%20return%20false%3B%0A%20%20%0A%20%20%0A%20%20return%20true%20%3B%0A%7D))%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

function bouncer(arr) {
  // Don't show a false ID to this bouncer.
  return arr;
}

bouncer([false, null, 0, NaN, undefined, ""].filter(function(value){
  return new Boolean(value) ;
}));

```

**Comment by jonathanihm** [comment]
The solution you provided is not correct just yet, but there is another issue you will need to handle before continuing.  You are applying the filter function on the test call of the bouncer method.  You should be calling the filter method on <code>arr</code> in the bouncer method.  Once you fix that, you should see better results (keep going!)

**Comment by raisedadead** [comment]
Hi, 

Thanks for reporting this issue. 

Here are some next steps that you should follow before we can help you with your query:
- Check [these troubleshooting steps](https://forum.freecodecamp.com/t/how-to-report-a-bug/19543) first.
- Please use the [Chat Rooms](https://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390) or try looking through our [forum](http://forum.freecodecamp.com) for help with a specific challenge and any coding assistance.

**IMPORTANT:** 
Please NOTE that this tracker is for reporting bugs and enhancements to freeCodeCamp's code base only.

Happy Coding!
